Bookish/Bookish.Web/Views/Home/BookPage.cshtml

```diff
@@ -31,8 +30,10 @@
                 <td>@book.BookName</td>
                 <td>@book.Author</td>
                 <td>@book.BarCode</td>
-                <td><button>here</button></td>
-</tr>
+                <td><form action="https://localhost:44394/home/bookinfo">
+                    <input name ="ISBN" type="submit" alt="Book info" value=@book.ISBN>
```

Comment:
It might be more natural to have the link be on the book name instead of ISBN? We also probably don't need a repeat of whichever field it is (at the moment it shows ISBN twice, once with the link and once without.)